server/utils/presenterUtils.ts

```diff
@@ -284,4 +286,34 @@ export default class PresenterUtils {
 
     return notFoundMessage
   }
+
+  static complexityLevelTagArgs(complexityLevel: ComplexityLevel): TagArgs {
+    const title = complexityLevel?.title.toUpperCase()
```

Comment:
I don't think you need the `?` here?

Reply:
ah good spot - this is probably left over from me moving the code from elsewhere